### ISSUE TITLE
High-Throughput Classification of Radiographs Using Deep Convolutional Neural Networks

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#156]
http://doi.org/10.1007/s10278-016-9914-9

> The study aimed to determine if computer vision techniques rooted in deep learning can use a small set of radiographs to perform clinically relevant image classification with high fidelity. One thousand eight hundred eighty-five chest radiographs on 909 patients obtained between January 2013 and July 2015 at our institution were retrieved and anonymized. The source images were manually annotated as frontal or lateral and randomly divided into training, validation, and test sets. Training and validation sets were augmented to over 150,000 images using standard image manipulations. We then pre-trained a series of deep convolutional networks based on the open-source GoogLeNet with various transformations of the open-source ImageNet (non-radiology) images. These trained networks were then fine-tuned using the original and augmented radiology images. The model with highest validation accuracy was applied to our institutional test set and a publicly available set. Accuracy was assessed by using the Youden Index to set a binary cutoff for frontal or lateral classification. This retrospective study was IRB approved prior to initiation. A network pre-trained on 1.2 million greyscale ImageNet images and fine-tuned on augmented radiographs was chosen. The binary classification method correctly classified 100 % (95 % CI 99.73–100 %) of both our test set and the publicly available images. Classification was rapid, at 38 images per second. A deep convolutional neural network created using non-radiological images, and an augmented set of radiographs is effective in highly accurate classification of chest radiograph view type and is a feasible, rapid method for high-throughput annotation.

**Comment by cgreene** [comment]
Data = ~1900 chest radiographs
Processing: Annotated as frontal/lateral. Divided into training/test/validation. Converted to 256x256 pixel images.
Independent test set: Downloaded public anonymized radiographs. Manually labeled each of 1000.

Training:
* Converted ImageNet to greyscale. Retrained the GoogLeNet model using greyscale. They also trained a model using a subset of image classes from five categories that they felt were most similar to the radiographs.
* Transferred the three (color ImageNet, greyscale ImageNet, and subset greyscale ImageNet) networks to their radiographs [implicitly to predict frontal/lateral?]. Fine tuning was performed on only the fully connected layers with only radiographs, only the fully connected layers with radiographs + other images, and all layers with radiographs and other images.
* From this, they predicted the probability of frontal and probability of lateral radiographs.

Results:
* Color ImageNet didn't do well unless they fine tuned all layers using the augmented set [radiographs + other images].
* Both greyscale methods did quite well either way [there are some significant p-values, but the performance measures are all within a percent of each other]

Take home: transferability comes up. Main summary is that due to limited data, some way to restrict the solution space via either more examples or more restrictions is important (e.g. greyscale *or* retraining with radiographs + other images).

**Comment by cgreene** [comment]
One important piece that's missing: what if you fine tuned all layers using only the radiographs. They never tried it due to the low sample number, but it's hard to draw a strong inference around their Color ImageNet results without that comparison.

**Comment by cgreene** [comment]
An issue with the evaluation is that the `validation set` changes. 

> For the models fine-tuned on original radiographs, validation sets consisted of 376 original radiographs. For models fine-tuned on augmented radiographs, validation sets consisted of 39,856 augmented radiographs

I think this makes it difficult to draw strong inferences from this paper. I'm going to close it. If someone else wants to discuss, feel free to re-open and incorporate into the review.
